src/components/Slide/player.css

```
@@ -7,54 +7,168 @@
+}
+
+.slide--mobile-device .slide-player__controls {
+  display: grid !important;
+  width: 100%;
+  padding: 0 20px !important;
+  grid-template-columns: repeat(
+    var(--slide-player-mobile-control-count, 4),
+    minmax(0, 1fr)
+  );
+  align-items: center;
+  padding: 0 20px;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)